Install `peerme` as executable & moved the entry_point into the module

andir opened the pull request.

andir committed:
Install `peerme` as executable & moved the entry_point into the module

cooperlees reviewed (changes requested):
Let's do 2 things:
- rename to main.py
- test templating still works, I'm worried about the relative lame pathing we've used (generate.py)

andir commented:
Your are right about the name.. just didn't come up with a good name at the time. i'll get on those changes in the morning :-)

andir commented:
I moved the templates into the `peerme` packaged and added them to the `MANIFEST.in` so they are being included in the installed package. Also the `peerme.commands` package was missing.

Running the `peerme` examples from /tmp/ with peerme in my path resulted in the correct output. I think this works as expected.

flokli commented on setup.py:
```diff
@@ -33,11 +33,15 @@
 except ImportError:
     from distutils.core import setup
 
+# since we've disutils support we can't use setuptools.find_packages for this
```
typo at 'disutils'

cooperlees reviewed (approved):
lgtm
